src/components/Form/SelectWidget.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+	FormControl,
+	Autocomplete,
+	TextField,
+	InputLabel,
+} from '@mui/material';
+import { WidgetProps } from '@rjsf/utils';
+
+export const SelectWidget = ({
+	id,
+	label,
+	value,
+	disabled,
+	name,
+	placeholder,
+	onChange,
+	multiple,
+	options,
+	required,
+}: WidgetProps) => {
+	const change = ({ target: { value } }: any) => {
+		return onChange(value === '' ? options.emptyValue : value);
+	};
+
+	const { enumOptions, enumDisabled } = options;
+	if (!Array.isArray(enumOptions)) {
+		return null;
+	}
+	const sanitizedEnumDisabled = Array.isArray(enumDisabled)
+		? enumDisabled
+		: null;
+	const selectOptions = enumOptions.map((option) => {
+		return {
+			...option,
+			disabled:
+				sanitizedEnumDisabled != null &&
+				sanitizedEnumDisabled.includes(option.value),
+		};
+	});
+
+	return (
+		<FormControl>
+			<InputLabel htmlFor={id}>
+				{label}
+				{required ? '*' : null}
```

Comment:
Not sure if I missed something but can't we use the `required` prop from `InputLabel` now that we're using that? 😕

Reply:
left over from the rebase, I should have checked better